server/hack/deploy.sh

```diff
@@ -48,16 +50,21 @@ else
 fi
 
 # updating config locally
-cd "${f}/config/default"
```

Comment:
is this intentional?

Reply:
Yes, we're already in the correct directory, no need to do an extra directory change.